pkg/flowaggregator/flowaggregator.go

```diff
@@ -683,3 +689,12 @@ func (fa *flowAggregator) updateFlowAggregator(opt *options.Options) {
 		}
 	}
 }
+
+func editEtcHost(ip string, hostname string) error {
```

Comment:
The K8s [documentation](https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/) recommends against editing this file directly.

Reply:
Deleted. I currently use `AppsV1().Deployments()` to update `hostAliases `.